CHANGELOG.md

```diff
@@ -10,45 +10,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### ⚠ BREAKING CHANGES
 
 - **list, list-item, list-item-group:** To know when `calcite-list-item` content is selected, listen to the event `calciteListItemSelect` instead of `click`.
-
```

Comment:
@benelan why are you removing this?

Reply:
This part should stay.

Reply:
Oh okay I thought it was accidently added like Eliza's was. I'll put it back